### PR TITLE
Uploading and viewing summaries

### PR DESCRIPTION
**Pull request by kokx**
Added the ability to upload and view summaries

### CODE REVIEW

File: module/Education/src/Education/Form/Upload.php [hunk 1]
```diff
@@ -4,9 +4,13 @@
...
 use Zend\Form\Form;
 use Zend\InputFilter\InputFilter;
+use Zend\InputFilter\InputFilterProviderInterface;
 use Zend\I18n\Translator\TranslatorInterface as Translator;
 
-class Upload extends Form
+/**
+ * Upload a summary
+ */
+class Upload extends Form implements InputFilterProviderInterface
```
Comment (stefanmajoor) [inline module/Education/src/Education/Form/Upload.php]: Why not call the file/class UploadSummary instead? That makes the comment superfluous

File: module/Education/src/Education/Service/Exam.php [hunk 2]
```diff
@@ -166,7 +166,7 @@ public function tempUpload($post, $files)
     public function upload($post, $files)
```
Comment (stefanmajoor) [inline module/Education/src/Education/Service/Exam.php]: why does the upload method in the Exam service upload a summary?

**Comment by stefanmajoor** [comment]
Why are there so many diffs where code for uploading exams is changed by code to upload summaries. Should exams no longer be uploaded?

**Comment by kokx** [comment]
There is a new way to upload exams. Which uploads exams in bulk (which is the case every quartile, when the education officer receives new exams of the exam weeks). Summaries, however, won't need to be uploaded in bulk.

File: module/Education/src/Education/Service/Exam.php (hunk 2 above)
Comment (kokx) [inline module/Education/src/Education/Service/Exam.php]: The naming may be improved here indeed. Will take a look at that.

File: module/Education/src/Education/Form/Upload.php (hunk 1 above)
Comment (kokx) [inline module/Education/src/Education/Form/Upload.php]: The naming may be improved here indeed. Will take a look at that.

**Comment by kokx** [comment]
Fixed all the previous problems. Can this be reviewed @jszanto or @stefanmajoor ?

**Pull request merged**